packages/core/src/sass/app/_common/layout/dashboard-dropdown.scss

```diff
@@ -0,0 +1,82 @@
+.dashboard-header-dropdown {
+    display: flex;
+    width: auto;
+    position: absolute;
+    padding: 2.2rem 0.8rem;
+    z-index: 99;
+    height: auto;
+    transform: translateY(7.2rem);
+    background-color: #ffffff;
```

Comment:
can we use css variables?

Reply:
Amended. Also for other properties involving colors in this file